posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -692,18 +692,17 @@ public class PostHog private constructor(
         if (!isEnabled()) {
             return defaultValue
         }
-        return featureFlags?.isFeatureEnabled(key, defaultValue) ?: defaultValue
+        val value = featureFlags?.isFeatureEnabled(key, defaultValue) ?: defaultValue
```

Comment:
in most SDKs isFeatureEnabled calls getFeatureFlag, so all of this handling can be in one place. Think we can do the same here?

Reply:
because of this tiny bit of code its trickier https://github.com/PostHog/posthog-android/blob/3a0ede85b9ed25dd993ff3944f74a7187dc175a7/posthog/src/main/java/com/posthog/internal/PostHogFeatureFlags.kt#L205-L214
since this is a compiled language and it has strongly typed methods, will get it merged for now and think if its worth the refactoring 🧐